apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -68,7 +60,7 @@ export const Login = ({ history }: LoginProps) => {
       setAccessToken(accessToken);
 
       try {
-        var decodedToken = jwtDecode(accessToken) as { id: number };
+        var decodedToken: IAuthToken = jwtDecode(accessToken);
       } catch (err) {
         console.error(err);
         updateToast(toast.TYPE.ERROR, `Received an invalid token`);
```

Comment:
Might be a good idea to move the string `Invalid Login Token` to a constant since it could be used in two places now

```suggestion
        updateToast(toast.TYPE.ERROR, `Invalid Login Token`);
```